themes/custom/droopler_theme/scss/components/_d_p_side_tiles.scss

```diff
@@ -25,9 +25,11 @@ $d-p-side-tiles-text-mobile-padding: 7% !default;
 $d-p-side-tiles-text-width: 35.625rem !default;
 $d-p-side-tiles-half-padding: 10% !default; // Spacing between content and half of the screen
 
-.d-p-side-tiles-wrapper {
+.theme-default .d-p-side-tiles-wrapper {
   background-color: $d-p-side-tiles-text-background-color;
+}
 
+.d-p-side-tiles-wrapper {
   &.image-side-left {
     background-color: $d-p-side-tiles-text-background-color-secondary;
   }
```

Comment:
Bartek, sprawdzałeś może jak to d-p-side-tiles zachowa sie w przypadku .theme-gray razem z  .image-side-left ?

zastanawiam się, czy to stylwoanie backgroundu dla .image-side-left nie powinno też trafić pod .theme-default.. ale dla innych .theme wtedy by trzeba bylo osobno dodać stylowanie dla .image-side-left

Reply:
sprawdze